api/src/main/java/ru/urururu/sanity/api/ParsersFacade.java

```diff
@@ -0,0 +1,49 @@
+package ru.urururu.sanity.api;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import ru.urururu.sanity.api.cfg.*;
+import ru.urururu.sanity.cpp.ValueParser;
+
+/**
+ * @author <a href="mailto:dmitriy.g.matveev@gmail.com">Dmitry Matveev</a>
+ */
+public abstract class ParsersFacade<T, V, I, B, Ctx extends CfgBuildingCtx<T, V, I, B, Ctx>> {
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Rename this generic name to match the regular expression '^[A-Z][0-9]?$'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS00119)